torch/utils/checkpoint.py

```diff
@@ -142,7 +142,7 @@ def backward(ctx, *args):
         return (None, None) + grads
 
 
-def checkpoint(function, *args, **kwargs):
+def checkpoint(function, *args, use_reentrant: bool = True, **kwargs):
```

Comment:
Maybe make it kwarg only? Or the fact that it is placed after `*args` already makes it kwarg only?

Reply:
Yep, it is kwarg only since it is after *args: https://www.vegardstikbakke.com/python-keyword-only/